src/vs/editor/browser/widget/codeEditorWidget.ts

```diff
@@ -1871,6 +1872,8 @@ export class CodeEditorWidget extends Disposable implements editorBrowser.ICodeE
 	}
 
 	private _detachModel(): ITextModel | null {
+		this._contributionsDisposable?.dispose();
```

Comment:
Is this called when the code editor widget itself gets disposed? Otherwise there might be a leak there.

Reply:
Ah, it is.